<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>508</height>
   </rect>
  </property>
  <widget class="QCheckBox" name="connectCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QGroupBox" name="controlGroupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>400</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="bassSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="balanceSlider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>61</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="sliderPosition">
     <number>30</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="muteCheckBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QSlider" name="trebleSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="logClearBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="trebleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Treble</string>
    </property>
   </widget>
   <widget class="QLabel" name="muteLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="bassLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Basss</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cmdValueText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
